
From: Valdis.Kletnieks@vt.edu

Small cleanup to make code safe for compiling with -Wundef..

Signed-off-by: valdis.kletnieks@vt.edu
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/kexec.h |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN include/linux/kexec.h~kexec-ifdef-cleanup include/linux/kexec.h
--- 25/include/linux/kexec.h~kexec-ifdef-cleanup	Wed Oct 13 15:11:07 2004
+++ 25-akpm/include/linux/kexec.h	Wed Oct 13 15:11:07 2004
@@ -1,7 +1,7 @@
 #ifndef LINUX_KEXEC_H
 #define LINUX_KEXEC_H
 
-#if CONFIG_KEXEC
+#ifdef CONFIG_KEXEC
 #include <linux/types.h>
 #include <linux/list.h>
 #include <asm/kexec.h>
_
